doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
+
+  Use a command similar to the following:
+
+        lxc config device add <instance_name> <device_name> nic nictype=<nic_type> ...
+
+  See {ref}`instance_device_type_nic` for a list of available NIC types and their configuration properties.
+
+  For example, you could add a pre-existing Linux bridge (`br0`) with the following command:
+
+        lxc config device add <instance_name> eth0 nic nictype=bridged parent=br0
+- {doc}`Create a managed network </howto/network_create>` and add it as a network device to your instance.
+  With this method, LXD has all required information about the configured network, and you only need to specify the network name when adding the device:
+
+        lxc config device add <instance_name> <device_name> nic network=<network_name>
+
+  If needed, you can add further properties to the command to override the default settings for the network.
+
+## Managed networks
+
+Managed networks in LXD are created and configured with the `lxc network` command.
+
+Depending on the network type, LXD either fully controls the network or just manages an external network interface.
+
+Note that not all {ref}`NIC types <instance_device_type_nic>` are supported as network types.
+LXD can only set up some of the types as managed networks.
+
+### Fully controlled networks
+
+Fully controlled networks provide most functionality, including, for example, the ability to do IP management.
+
 LXD supports the following network types:
 
- - [bridge](#network-bridge): Creates an L2 bridge for connecting instances to (can provide local DHCP and DNS). This is the default.
- - [macvlan](#network-macvlan): Provides preset configuration to use when connecting instances to a parent macvlan interface.
- - [sriov](#network-sriov): Provides preset configuration to use when connecting instances to a parent SR-IOV interface.
- - [ovn](#network-ovn): Creates a logical network using the OVN software defined networking system.
- - [physical](#network-physical): Provides preset configuration to use when connecting OVN networks to a parent interface.
+{ref}`network-bridge`
+: A network bridge creates a connection between different networks or network segments.
+
+  In LXD context, the `bridge` network type creates an L2 bridge that connects the instances that use it and makes it possible to pass traffic between them.
+  The bridge can also provide local DHCP and DNS.
+
+  This is the default network type.
+
+{ref}`network-ovn`
+: {abbr}`OVN (Open Virtual Network)` is a software-defined networking system that supports virtual network abstraction.
+  You can use it to build your own private cloud.
+  See https://www.ovn.org/ for more information.
+
+  In LXD context, the `ovn` network type creates a logical network.
+  To set it up, you must install and configure the OVN tools.
+  In addition, you must create an uplink network that provides the network connection for OVN.
+  As the uplink network, you should use one of the external network types.
```

Comment:
Not true, as the ovn setup example shows, for single-server setups you can use managed bridge network types as uplinks too.

Reply:
Is this why you didn't use the word "must"? :)

Reply:
The reason the example shows using a managed LXD bridge is that it lowers the requirements for getting started with OVN networks.

Reply:
LGTM